### PR TITLE
Fixes #42: seed the database from yarn!

### LINKED ISSUE #42: consolidate migrate/seeding of database to a yarn script
It would be nice to have an installation of postgres as well as database seeding and migration from npm scripts.

for example:
 `yarn run seed` 

```bash
$ createdb react_twitter_dev
$ createdb react_twitter_test
$ cd server
$ knex migrate:latest
$ knex migrate:latest --env test
$ knex seed:run
$ knex seed:run --env test
$ cd ../
```

> Make sure you have postgres installed globally, then

Installing postgres automagically may be a challenge if we wanted to stay cross platform... could use `brew`, and apt or yum where available.  Windows would be left out in the cold.  Is that such a big deal though? 😝

### PR DESCRIPTION
**Pull request by garyspencer**
The seed script does not work once the databases are created.  Should we add something to drop those tables so a re-seed is successful?

### CODE REVIEW

**Comment by garyspencer** [comment]
I need to get in line with the yarniverse, making some edits now

**Comment by garyspencer** [comment]
Wasn't sure if I should have included yarn.lock with my commit... this was after a fresh `yarn install` after I added a devdependency

File: package.json [hunk 1]
```diff
@@ -24,9 +24,11 @@
     "server": "nodemon server/index.js",
     "build": "react-scripts build",
     "test": "cd server/; NODE_ENV=test mocha --exit",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "seed": "dropdb react_twitter_dev && dropdb react_twitter_test && createdb react_twitter_dev && createdb react_twitter_test && cd server &&knex migrate:latest && knex migrate:latest --env test && knex seed:run && knex seed:run --env test"
```
[the remark this answers is not in the document]
Comment (garyspencer) [reply package.json]: on it!

**Comment by jenniferwiner** [comment]
@garyspencer Can you pull the latest master into your branch and then I'll merge this in? Thanks!!!!!!!!

**Pull request merged**